front/pages/api/w/[wId]/assistant/builder/suggestions.ts

```diff
@@ -120,7 +121,27 @@ async function handler(
           },
         });
       }
-      const suggestions = responseValidation.right;
+      const suggestions = responseValidation.right as {
+        status: "ok";
+        suggestions: string[] | null | undefined;
+      };
+      if (suggestionsType === "name") {
+        // Filter out suggested names that are already in use in the workspace.
+        const existingNames = (
+          await AgentConfiguration.findAll({
```

Comment:
We try to avoid calling models directly in API routes, see this rule: https://www.notion.so/dust-tt/Engineering-e0f834b5be5a43569baaf76e9c41adf2?pvs=4#29c353e68cff465d93824c8473a19399

Could you move this logic in a function on lib/api? 🙏🏻

Reply:
Otherwise logic look good 👍🏻